Python/compile.c

```diff
@@ -4049,6 +4049,34 @@ maybe_optimize_method_call(struct compiler *c, expr_ty e)
     return 1;
 }
 
+static int
+validate_keywords(struct compiler *c, asdl_seq* keywords) {
+    int nkeywords = asdl_seq_LEN(keywords);
+    for (int i = 0; i < nkeywords; i++) {
+        keyword_ty key = ((keyword_ty)asdl_seq_GET(keywords, i));
+        if (key->arg == NULL) {
+            continue;
+        }
+        for (int j = 0; j < nkeywords; j++) {
+            if (i == j) {
+                continue;
+            }
```

Comment:
Couldn't this just be `for (int j = i+1; j < keywords; j++)`? This way we could also omit the `if (i ==j)` check.

Reply:
Done!